config/routes.rb

```diff
@@ -144,8 +147,8 @@
       end
     end
     resources :users
-
-    get '/rescan' => 'rescan#show'
-    post '/rescan' => 'rescan#start'
+    resources :rescans, only: %i[index show]
+    post 'rescans/:id', to: 'rescans#start'
+    post 'rescans', to: 'rescans#start_all'
```

Comment:
I feel like it should be possible to express this a bit more cleanly in the DSL.

Reply:
There might be, but I couldn't find it (at least if we want these paths). 
If we want to add these routes as extra actions on the resource, we'll always end up with something like this:

```ruby
resources :rescans, only: %i[index show] do
    post 'start', on: :member
    post 'start_all', on: :collection 
end

# start_rescan POST   /api/rescans/:id/start(.:format)                                                                  rescans#start
# start_all_rescans POST   /api/rescans/start_all(.:format)                                                                  rescans#start_all
```